runtime/rasdump/javadump.cpp

```diff
@@ -3378,6 +3377,32 @@ JavaCoreDumpWriter::writeTrailer(void)
 {
 	_OutputStream.writeCharacters(
 		"0SECTION       Javadump End section\n"
+		"NULL           =================================\n"
+	);
+
+	PORT_ACCESS_FROM_JAVAVM(_VirtualMachine);
+	OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
+
+	/* Write the date and time that the dump completed generation. */
+	char timeStamp[_MaximumTimeStampLength + 1];
+	int64_t now = j9time_current_time_millis();
+
+	omrstr_ftime_ex(timeStamp, _MaximumTimeStampLength, TIMESTAMP_FORMAT, now, OMRSTR_FTIME_FLAG_LOCAL);
+	timeStamp[_MaximumTimeStampLength] = '\0';
+
+	_OutputStream.writeCharacters("1TIDMPCOMPLETE Dump completed: ");
+	_OutputStream.writeCharacters(timeStamp);
+	_OutputStream.writeInteger(now % 1000, ":%03d"); /* Add the milliseconds. */
+	_OutputStream.writeCharacters("\n");
+
+	/* Calculate the approximate time it took to produce the dump. */
+	int64_t duration = now - _DumpStart;
```

Comment:
I suggest this should use `uint64_t` (or `U_64`) with an explicit cast, for compatibility with its use in the call to `writeInteger64()`:
```
	uint64_t duration = (uint64_t)(now - _DumpStart);
```

Reply:
Thanks, commit added